scripts/tcs_startup.sh

```diff
@@ -106,6 +106,10 @@ stop_avalon_components_forcefully()
     ps -ef | grep bin/$KV_STORAGE | grep -v grep | awk '{print $2}' | xargs -r kill -9;
     ps -ef | grep $ENCLAVE_MANAGER | grep -v grep | awk '{print $2}' | xargs -r kill -9;
     ps -ef | grep defunct | grep -v grep | cut -b8-20 | xargs -r kill -9
+    lsof -t -i:5555 | xargs -r kill -9
+    lsof -t -i:9090 | xargs -r kill -9
+    lsof -t -i:1947 | xargs -r kill -9
```

Comment:
Please extract the port number from URL to avoid hard coding at multiple places

Reply:
Done